packages/gafl-webapp-service/src/pages/summary/licence-summary/route.js

```diff
@@ -77,13 +78,15 @@ export const getData = async request => {
     cost: permission.permit.cost,
     birthDateStr: moment(permission.licensee.birthDate, cacheDateFormat).format('Do MMMM YYYY'),
     uri: {
-      name: NAME.uri,
-      licenceLength: LICENCE_LENGTH.uri,
-      licenceType: LICENCE_TYPE.uri,
-      licenceToStart: LICENCE_TO_START.uri,
-      dateOfBirth: DATE_OF_BIRTH.uri,
-      disabilityConcession: DISABILITY_CONCESSION.uri,
-      licenceStartDate: permission.isRenewal ? RENEWAL_START_DATE.uri : LICENCE_TO_START.uri,
+      name: addLanguageCodeToUri(request, NAME.uri),
+      licenceLength: addLanguageCodeToUri(request, LICENCE_LENGTH.uri),
+      licenceType: addLanguageCodeToUri(request, LICENCE_TYPE.uri),
+      licenceToStart: addLanguageCodeToUri(request, LICENCE_TO_START.uri),
+      dateOfBirth: addLanguageCodeToUri(request, DATE_OF_BIRTH.uri),
+      disabilityConcession: addLanguageCodeToUri(request, DISABILITY_CONCESSION.uri),
```

Comment:
Same goes here - test that the values are the ones returned by addLanguageCodeToUri.